projects/storefrontlib/src/cms-components/user/login-form/login-form.component.ts

```diff
@@ -43,7 +44,8 @@ export class LoginFormComponent implements OnInit, OnDestroy {
     private globalMessageService: GlobalMessageService,
     private fb: FormBuilder,
     private authRedirectService: AuthRedirectService,
-    private winRef?: WindowRef
+    private winRef?: WindowRef,
```

Comment:
we have to also check any occurence of `winRef` in the code, since it became optional. also deprecate this check from 1.3